src/modules/store.js

```diff
@@ -0,0 +1,63 @@
+export default class Store {
+  static getList() {
+    let list = [];
+    if (localStorage.getItem('task') === null) {
+      list = [];
+    } else {
+      list = JSON.parse(localStorage.getItem('task'));
+    }
+    return list;
+  }
+
+  static add(task) {
+    const list = Store.getList();
+    list.push(task);
+    Store.updateIndex();
+    localStorage.setItem('task', JSON.stringify(list));
+  }
+
+  static remove(index) {
+    const list = Store.getList();
+    for (let i = 0; i < list.length; i += 1) {
+      if (list[i].index === index) {
+        list.splice(index - 1, 1);
+      }
+    }
+    localStorage.setItem('task', JSON.stringify(list));
+  }
+
+  static updateIndex() {
+    const list = Store.getList();
+    for (let j = 0; j < list.length; j += 1) {
+      list[j].index = j + 1;
+    }
+    localStorage.setItem('task', JSON.stringify(list));
```

Comment:
You have repeatedly used this code to improve your DRY, you could write a save() method to execute the same line